### PR TITLE
task/close_1.6.0

### PR DESCRIPTION
**Pull request by frbattid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Other interesting links are:
 
 * Our [Apiary Documentation](http://telefonicaid.github.io/fiware-cygnus/api/latest) if you want to know how to use our API methods for Cygnus.
 * [cygnus-ngsi](doc/cygnus-ngsi/integration) **integration** examples .
-* [cygnus-ngsi](https://edu.fiware.org/mod/resource/view.php?id=1006) **introductory course** in FIWARE Academy.
+* [cygnus-ngsi](https://edu.fiware.org/mod/resource/view.php?id=1015) **introductory course** in FIWARE Academy.
```
Comment (fgalan) [inline README.md]: Is this really part of release closing of a king of "piggybacked bugfix" in this PR?

File: cygnus-common/spec/SPECS/cygnus-common.spec [hunk 2]
```diff
@@ -169,6 +169,9 @@ rm -rf $RPM_BUILD_ROOT
 /var/run/%{_project_name}
 
 %changelog
+* Fri Dec 02 2016 Francisco Romero <francisco.romerobueno@telefonica.com> 1.6.0
+- [cygnus][hardening] Add IPv6 support to API and GUI (#1261)
```
Comment (fgalan) [inline cygnus-common/spec/SPECS/cygnus-common.spec]: Only one new line in changelog... however in CNR 25 lines has been removed...

Comment (fgalan) [inline cygnus-common/spec/SPECS/cygnus-common.spec]: I have just seen https://github.com/telefonicaid/fiware-cygnus/pull/1337/files#diff-09eccda12e4412bcaf14f1e078a741a9R100 below... Thus, I understand this is correct.

File: doc/index.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ Other interesting links are:
 
 * Our [Apiary Documentation](http://telefonicaid.github.io/fiware-cygnus/api/latest) if you want to know how to use our API methods for Cygnus.
 * [cygnus-ngsi](doc/cygnus-ngsi/integration) **integration** examples .
-* [cygnus-ngsi](https://edu.fiware.org/mod/resource/view.php?id=1006) **introductory course** in FIWARE Academy.
+* [cygnus-ngsi](https://edu.fiware.org/mod/resource/view.php?id=1015) **introductory course** in FIWARE Academy.
```
Comment (fgalan) [inline doc/index.md]: Same here.

File: README.md (hunk 1 above)
Comment (frbattid) [inline README.md]: It is an update of the PPT link, in this case poiting to 1.6.0 instead of 1.5.0

File: doc/index.md (hunk 3 above)
Comment (frbattid) [inline doc/index.md]: It is an update of the PPT link, in this case poiting to 1.6.0 instead of 1.5.0

File: README.md (hunk 1 above)
Comment (fgalan) [inline README.md]: NTC

File: cygnus-common/spec/SPECS/cygnus-common.spec (hunk 2 above)
Comment (fgalan) [inline cygnus-common/spec/SPECS/cygnus-common.spec]: NTC

File: doc/index.md (hunk 3 above)
Comment (fgalan) [inline doc/index.md]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**